Mathlib/CategoryTheory/Category/Preorder.lean

```diff
@@ -92,27 +92,27 @@ alias leOfHom ← _root_.Quiver.Hom.le
 
 -- porting note: linter seems to be wrong here
 @[simp, nolint simpNF]
```

Comment:
Why do we want to simplify a PROOF? We have proof irrelevance.

Reply:
Please remove `simp` and/or modify the porting note. Otherwise LGTM. Thanks!

bors d+